src/components/Clients/ClientListTableRow.jsx

```diff
@@ -95,7 +95,7 @@ const ClientListTableRow = ({ labelId, client, state, dispatch }) => {
           rel="noreferrer"
           style={{ textDecoration: 'none', color: theme.palette.primary.dark }}
         >
-          {client.webId}
+          Link to Pod Profile
```

Comment:
Fine with this for now, but I'd prefer the name is just a link to this - not another whole column like now.  But, that's for another PR.

Reply:
Either way, we may want to add an "open in new tab" icon so the user knows it'll exit the app.

e.g.
`import OpenInNewIcon from '@mui/icons-material/OpenInNew';`